jbpm-test-coverage/pom.xml

```diff
@@ -125,6 +125,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>xmlunit</groupId>
+      <artifactId>xmlunit</artifactId>      
+      <scope>test</scope>
+    </dependency>
```

Comment:
this is not longer used, you can remove it

Reply:
Sure Gonzalo, will make the changes accordingly.. Thank you!